### PR TITLE
added support for user defined tags on generated `container_import` r…

### PR DESCRIPTION
**Pull request by sha1n**
@nlopezgi this is my proposed fix for #764 
I tested this with `--experimental_allow_tags_propagation` and it works as expected. Without the flag, the `GUNZIP` and `GZIP` actions will run against remote-cache. But this is expected and this PR solves a more generic case and is not specific to remote-caching.

**A few notes**:
- I'm not so sure about the name of the attribute on the `container_pull` repo rule.. will be happy to hear other suggestions.
- I am unable to build/test this repository on my MacOS environment, is there any special setup I need to do?
- What about testing? Can you give me some pointers to something I can follow in order to test this?

### CODE REVIEW

File: container/pull.bzl [hunk 1]
```diff
@@ -80,24 +80,31 @@ _container_pull_attrs = {
         doc = "(optional) The tag of the image, default to 'latest' " +
               "if this and 'digest' remain unspecified.",
     ),
+    "import_tags": attr.string_list(
```
Comment (nlopezgi) [inline container/pull.bzl]: These must be in alphabetical order (name is fine btw)

Comment (sha1n) [reply container/pull.bzl]: Fixed.

**Comment by sha1n** [comment]
> So can you just verify this is working as intended for you and then we can merge
@nlopezgi I tested this locally with the `no-remote-cache` tag + `--experimental_allow_tags_propagation` - works as intended.

**Comment by nlopezgi** [comment]
Can you please run buildifier on container/pull.bzl and update your branch? Should be good to go once the buildkite presubmit passes; I can then run gcbtests and approve and merge if everything passes.

**Comment by sha1n** [comment]
Thanks @nlopezgi. Done! 

The build seems to fail on an unrelated issue. I think the cause is:
```
(06:21:20) ERROR: An error occurred during the fetch of repository 'zlib':
   java.io.IOException: Error downloading [https://zlib.net/zlib-1.2.11.tar.gz] to /var/lib/buildkite-agent/.cache/bazel/_bazel_buildkite-agent/ec321eb2cc2d0f8f91b676b6d4c66c29/external/zlib/zlib-1.2.11.tar.gz: connect timed out
(06:21:20) ERROR: /var/lib/buildkite-agent/.cache/bazel/_bazel_buildkite-agent/ec321eb2cc2d0f8f91b676b6d4c66c29/external/com_google_protobuf/BUILD:150:1: @com_google_protobuf//:protobuf depends on @zlib//:zlib in repository @zlib which failed to fetch. no such package '@zlib//': java.io.IOException: Error downloading [https://zlib.net/zlib-1.2.11.tar.gz] to /var/lib/buildkite-agent/.cache/bazel/_bazel_buildkite-agent/ec321eb2cc2d0f8f91b676b6d4c66c29/external/zlib/zlib-1.2.11.tar.gz: connect timed out
```

**Comment by nlopezgi** [comment]
> Thanks @nlopezgi. Done!
> 
> The build seems to fail on an unrelated issue. I think the cause is:
> 
> ```
> (06:21:20) ERROR: An error occurred during the fetch of repository 'zlib':
>    java.io.IOException: Error downloading [https://zlib.net/zlib-1.2.11.tar.gz] to /var/lib/buildkite-agent/.cache/bazel/_bazel_buildkite-agent/ec321eb2cc2d0f8f91b676b6d4c66c29/external/zlib/zlib-1.2.11.tar.gz: connect timed out
> (06:21:20) ERROR: /var/lib/buildkite-agent/.cache/bazel/_bazel_buildkite-agent/ec321eb2cc2d0f8f91b676b6d4c66c29/external/com_google_protobuf/BUILD:150:1: @com_google_protobuf//:protobuf depends on @zlib//:zlib in repository @zlib which failed to fetch. no such package '@zlib//': java.io.IOException: Error downloading [https://zlib.net/zlib-1.2.11.tar.gz] to /var/lib/buildkite-agent/.cache/bazel/_bazel_buildkite-agent/ec321eb2cc2d0f8f91b676b6d4c66c29/external/zlib/zlib-1.2.11.tar.gz: connect timed out
> ```

Yeah there's, an infra issue with our CI atm, I'll re run once that is resolved.

**Comment by nlopezgi** [comment]
Updating branch to see if Buildkite issue is now resolved

**Pull request merged**